examples/demo.js

```diff
@@ -116,6 +116,9 @@ window.onload = function () {
     else if (dropDownValue == 'save-seq') {
       saveSequence();
     }
+    else if (dropDownValue == 'save-to-publiclab.org' ){
+      savetopubliclab();
+    }
```

Comment:
Two things here: 
1. The condition of the if statement has a linting issue. It should be `else if (dropDownValue == 'save-to-publiclab.org') {`.
2. The function name should be in camel case: saveToPublicLab